pygfx/utils/__init__.py

```diff
@@ -1,3 +1,21 @@
+"""
+Utility functions for pygfx.
+
+.. autosummary::
+    :toctree: utils/
+    :template: ../_templates/clean_class.rst
+
+    Color
+    load.load_scene
+    show.show
+    show.Display
+    viewport.Viewport
+    text.font_manager
+    cm
```

Comment:
In the docs there is also a `functions` section?  https://pygfx--430.org.readthedocs.build/en/430/_autosummary/pygfx.utils.html

Reply:
Also the docs on e.g. `cm` speak of an alias: https://pygfx--430.org.readthedocs.build/en/430/_autosummary/utils/pygfx.utils.cm.html#pygfx.utils.cm

Reply:
> In the docs there is also a functions section?

True, looks like the default template adds a block for functions defined on a module. I'll have a look and remove them.

> Also the docs on e.g. cm speak of an alias:

Good catch. this should be a neatly rendered page of a sub-module.

Reply:
fixed.